wallet-new/src/Cardano/Wallet/API/V1/LegacyHandlers/Transactions.hs

```diff
@@ -119,9 +101,48 @@ estimateFees Payment{..} = do
     cAccountId <- migrate pmtSource
     utxo <- V0.getMoneySourceUtxo ws (V0.AccountMoneySource cAccountId)
     outputs <- V0.coinDistrToOutputs =<< mapM migrate pmtDestinations
-    efee <- V0.runTxCreator policy (V0.computeTxFee pendingAddrs utxo outputs)
-    case efee of
-        Right fee ->
-            single <$> migrate fee
-        Left err ->
-            throwM (convertTxError err)
+    fee <- V0.rewrapTxError "Cannot compute transaction fee" $
+        eitherToThrow =<< V0.runTxCreator policy (V0.computeTxFee pendingAddrs utxo outputs)
+    single <$> migrate fee
+
+
+newUnsignedTransaction
+    :: forall ctx m . (V0.MonadWalletTxFull ctx m)
+    => Payment
+    -> m (WalletResponse RawTransaction)
+newUnsignedTransaction pmt@Payment {..} = do
+    -- We're creating new transaction as usually, but we mustn't sign/publish it.
+    -- This transaction will be signed on the client-side (mobile client or
+    -- hardware wallet), and after that transaction (with its signature) will be
+    -- sent to backend.
+    batchPayment <- createBatchPayment pmt
+    tx <- V0.newUnsignedTransaction batchPayment
+    let txInHexFormat = B16.encode $ serialize' tx
+        rawTx = RawTransaction txInHexFormat
+    pure $ single rawTx
+
+
+newSignedTransaction
+    :: forall ctx m . (V0.MonadWalletTxFull ctx m)
+    => (TxAux -> m Bool)
+    -> SignedTransaction
+    -> m (WalletResponse Transaction)
+newSignedTransaction _ SignedTransaction {..} = do
+    -- It is assumed that we received a transaction which was signed on the
+    -- client side (mobile client or hardware wallet).
+    -- Now we have to submit it as usually.
+    --
+    -- It's just a stub instead of 'undefined'.
+    fakeTransaction <- liftIO $ generate arbitrary
+    pure $ single fakeTransaction
```

Comment:
This implementation needs to be done still

Reply:
Yes, I'm working on it right now. :-)